README.rst

```diff
@@ -37,7 +37,7 @@ Features:
 * save high-resolution image
 * works even with a large number of photos (> 100)
 * integrates into the GNOME environment
-* available in English, French, German, Czech, Italian, Bulgarian, Dutch and Russian
+* available in English, Bulgarian, Czech, Dutch, French, German, Italian, Russian and Spanish
```

Comment:
Can you keep the same order and just append Spanish?

Reply:
Ok. But when there are more and more supported languages, it is easier to find own in the list, when it sorted in alphabetical order.

Reply:
Sure, but I think we'll stop adding languages here at some point.

Reply:
When will the new version be released in distributions like Ubuntu, Linux Mint and other? How long is this process?

Reply:
@adrienverge, I can't run the program with the Russian language according to the "Hacking" instructions. Always runs with English. Is the instruction up-to-date?

Reply:
> When will the new version be released in distributions like Ubuntu, Linux Mint and other? How long is this process?

I don't know, I don't maintain these distributions packages. I can make a new release in the GitHub repo, if needed.

> I can't run the program with the Russian language according to the "Hacking" instructions. Always runs with English. Is the instruction up-to-date?

I don't know. You'll have to debug.

Reply:
The new release would be great!

Reply:
> The new release would be great!


Done: version 1.4.5 released :+1: